sdk/src/wallet/operations/transaction/high_level/staking/extend.rs

```diff
@@ -83,8 +83,8 @@ where
                 past_bounded_epoch,
                 end_epoch,
             ));
-            options.custom_inputs = Some(vec![account_output_data.output_id]);
-            options.context_inputs = Some(vec![ContextInput::from(RewardContextInput::new(0)?)]);
+            options.mandatory_inputs = [account_output_data.output_id].into();
```

Comment:
Should we not set the flag to false then?

Reply:
No, because it's fine for ISA to select more inputs

Reply:
I thought `custom_inputs` meant no additional inputs

Reply:
It did at one point. Now the flag controls that instead